src/schema/types/models/User.ts

```diff
@@ -1,54 +1,69 @@
-import { PrismaClient } from "@prisma/client";
-import { objectType } from "nexus";
+import { interfaceType, objectType } from "nexus";
 
-import { PortfolioService, ProjectService, UserService } from "service";
-
-const prisma = new PrismaClient();
+import { PortfolioService, ProjectService } from "service";
 
 export const User = objectType({
-  name: "User",
-  definition(t) {
-    t.int("id");
-    t.string("name");
-    t.string("email");
-    t.string("school");
-    t.string("profile_image");
-    t.string("introduction");
-    t.int("entrance_year");
-    t.int("grade", {
-      resolve: (root) => {
-        return new Date().getFullYear() - root.entrance_year;
-      },
-    });
-    t.field("portfolio", {
-      type: "Portfolio",
-      resolve: async (root, _, __) => {
-        return await PortfolioService.getPortfolio(root.id);
-      },
-    });
-    t.list.field("projects", {
-      type: "Project",
-      resolve: async (root, _, __) => {
-        return await ProjectService.getOwnProjectsOfUser(root.id);
-      },
-    });
-    t.list.field("participated_projects", {
-      type: "Project",
-      resolve: async (root, _, __) => {
-        return await ProjectService.getParticipatedProjectsOfUser(root.id);
-      },
-    });
-    t.list.field("liked_projects", {
-      type: "Project",
-      resolve: async (root, _, __) => {
-        return await ProjectService.getLikedProjectsOfUser(root.id);
-      },
-    });
-    t.list.field("liked_portfolios", {
-      type: "Portfolio",
-      resolve: async (root, _, __) => {
-        return await PortfolioService.getLikedPortfoliosOfUser(root.id);
-      },
-    });
-  },
+    name: "User",
+    definition(t) {
+        t.int("id");
+        t.string("name");
+        t.string("email");
+        t.string("school");
+        t.string("profile_image");
+        t.string("introduction");
+        t.int("entrance_year");
+        t.int("grade", {
+            resolve: (root) => {
```

Comment:
```suggestion
            resolve: (root, _, __) => {
```
나머지 인자값도 들어오지만, 사용하지않는다는 의미로 이렇게 적어주는 것이 좋다고 생각합니다.

Reply:
아 구별하기 더 쉽겠네요 좋습니다😃😃